### ISSUE TITLE
Verifica SVM con kernel 'sigmoid'

### ISSUE THREAD

**Issue by mz1991** [mz1991/Bayes-vs-SVM#1]
L'accuratezza della SVM con kernel sigmoid è molto bassa. ( 55%)
Verificare modificando qualche parametro in fase di creazione SVM.

sklearn.svm.SVC:
http://scikit-learn.org/stable/modules/generated/sklearn.svm.SVC.html#sklearn.svm.SVC

**Comment by mz1991** [comment]
Aggiunti parametri `gamma=1/float(len(dataCopy)),C=100` in fase di creazione _SVM object_.
